index.html

```diff
@@ -1,225 +1,240 @@
-          <ul>
-            <li>
-              <a href="#"><img src="/images/twitter.png" alt=""></a>
-            </li>
-            <li>
-              <a href="#"><img src="/images/Linkedin-icon.png" alt=""></a>
-            </li>
-            <li>
-              <a href="#"><img src="/images/medium.png" alt=""></a>
-            </li>
-            <li>
-              <a href="#"><img src="/images/github.png" alt=""></a>
-            </li>
-            <li>
-              <a href="#"><img src="/images/angellist.png" alt=""></a>
-            </li>
-          </ul>
-          <a class="down-icon" href="#">
-            <img src="/images/down.png" alt="down-icon">
-          </a>
-        </div>
-      </section>
-      <section class="works">
-        <h2 class="section-title">
-          My Recent Works
-        </h2>
-        <div class="card-container">
-          <div class="card">
-            <div class="card-img">
-            </div>
-            <div class="card-content">
-              <h4 class="p-title"> 
-                Multi-Post Stories Gain+Glory 
-              </h4>
-              <ul class="tags">
-                <li>Ruby on rails</li>
-                <li>css</li>
-                <li>JavaScript</li>
-                <li>html</li>
-              </ul>
-              <button type="button" class="primary-btn">
-                See Project
-              </button>
-            </div>
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Marco Oquendo</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <header>
+    <span class="welcome">Welcome</span>
+    <nav>
+      <a href="#"><img src="/images/Icon-Menu.png" alt="hamburger-icon"></a>
+    </nav>
+  </header>
+  <main>
+    <section class="headline">
+      <h1 class="title name">Hey there. I'm Marco</h1>
+      <h2 class="title ocupation">I'm a software developer</h2>
+      <p class="description">
+        I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+      <div class="social-container">
+        <ul>
+          <li><a href="#"><img src="/images/twitter.png" alt=""></a></li>
+          <li><a href="#"><img src="/images/Linkedin-icon.png" alt=""></a></li>
+          <li><a href="#"><img src="/images/medium.png" alt=""></a></li>
+          <li><a href="#"><img src="/images/github.png" alt=""></a></li>
+          <li><a href="#"><img src="/images/angellist.png" alt=""></a></li>
+        </ul>
+      </div>
+      <a class="down-icon" href="#">
+        <img src="/images/down.png" alt="down-icon">
+      </a>
+    </section>
+    <section class="work-container">
+      <h2>
+        My Recent Works
+      </h2>
+      <hr>
+      <div class="work-grid">
```

Comment:
- [ ] Kindly ensure that your cards are in 2 rows with 3 projects each (6 projects in total), currently this is only achieved in `1440px` screensize and above, please make sure that screensizes from `769px` where your media query starts has this implemented 👍🏿